basic_algs/src/main.rs

```diff
@@ -0,0 +1,94 @@
+fn max_out_of_10(arr: &[i64; 10]) -> i64 {
+    let mut mx = arr[0];
+    for val in arr {
+        if mx < *val {
+            mx = *val;
+        }
+    }
+    return mx;
```

Comment:
Здесь и далее можно не писать `return`:
```suggestion
    mx
```
Считается, что в этом случае код "более идиоматичный". Никаких практических недостатков у такого кода, впрочем, нет. В инструменте clippy есть линт для таких случаев: https://rust-lang.github.io/rust-clippy/master/#needless_return